sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2048,6 +2049,40 @@ SIREPO.viewLogic('sourceView', function(appState, panelState, $scope) {
     ];
 });
 
+SIREPO.viewLogic('tallyView', function(appState, panelState, $scope) {
+
+    const ALL_TYPES = SIREPO.APP_SCHEMA.enum.TallyFilter
+        .map(x => x[SIREPO.ENUM_INDEX_VALUE]);
+    const inds = SIREPO.UTILS.indexArray(SIREPO.APP_SCHEMA.constants.maxFilters, 1);
+    const TYPE_NONE = 'None';
+
+    $scope.modelData = appState.models[$scope.modelName];
+
+    function type(index) {
+        return $scope.modelData[`filter${index}`]._type;
+    }
+
```

Comment:
Here, you don't want to have $scope.modelData, or set it to something when the directive is created. Otherwise it will always be set to the first model edited (in case there are more than one tally in the simulation).

Instead, type() should directly use appState to get the current model values:
```javascript
    function type(index) {
        return appState.models[`filter${index}`]._type;
    }
```

Reply:
[0baffb8](https://github.com/radiasoft/sirepo/pull/6463/commits/0baffb8cc4966aeb6d6fea035cce4806ec19b0f5)